src/common/constants.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export const COMMON_CONSTANTS = {
```

Comment:
Does it make sense for initializers to need to provide the database and store names? Seems like this is an implementation detail that could be hidden from consumers, perhaps in a function that wraps the `Store` constructor?

Reply:
Updated.